app/javascript/packs/webShare.js

```diff
@@ -0,0 +1,2 @@
+/* eslint no-unused-vars: ["error", { "argsIgnorePattern": "_" }] */
+import 'web-share-wrapper';
```

Comment:
Hmm, I kind of agree with this. Except unpkg describes its module support as experimental and the module version itself is non-minified.

Also, aside from your suggestion in #1579, unpkg isnt used anywhere else. So using unpkg would require a new HTTP connection to be set up. The other scripts on the page are served from Fastly's CDN, which we already have a connection to (probably an H2 connection) so there's no need to reconnect and it can be downloaded in parallel.

I like the point about fully deferring it though. I've moved the pack tag to the bottom of the `articles/show` view, since that's the only page the component is used on.